# HG changeset patch
# User Rik <rik@octave.org>
# Date 1473896745 25200
#      Wed Sep 14 16:45:45 2016 -0700
# Node ID 49103c6e0782e640c56090faf953f45625260ccc
# Parent  93ea313301f948a2a6eac38e885d5bd971ae5440
Enable AlphaChannel in Qt GLCanvas.

* GLCanvas.cc (GLCanvas::GLCanvas): Add QGL::AlphaChannel to format used in
constructor.

diff --git a/libgui/graphics/GLCanvas.cc b/libgui/graphics/GLCanvas.cc
--- a/libgui/graphics/GLCanvas.cc
+++ b/libgui/graphics/GLCanvas.cc
@@ -30,20 +30,24 @@ along with Octave; see the file COPYING.
 #include "GLCanvas.h"
 #include "gl-select.h"
 
 namespace QtHandles
 {
 
   GLCanvas::GLCanvas (QWidget* xparent, const graphics_handle& gh)
 #if defined (Q_OS_WIN32)
-    : QGLWidget (QGLFormat (QGL::SampleBuffers | QGL::IndirectRendering),
-                 xparent), Canvas (gh)
+    : QGLWidget (QGLFormat (QGL::SampleBuffers | QGL::AlphaChannel |
+                            QGL::IndirectRendering),
+                 xparent),
+      Canvas (gh)
 #else
-    : QGLWidget (QGLFormat (QGL::SampleBuffers), xparent), Canvas (gh)
+    : QGLWidget (QGLFormat (QGL::SampleBuffers | QGL::AlphaChannel),
+                 xparent),
+      Canvas (gh)
 #endif
   {
     setFocusPolicy (Qt::ClickFocus);
     setFocus();
   }
 
   GLCanvas::~GLCanvas (void)
   { }
